README.md

```diff
@@ -1,2 +1,72 @@
-# flock
-Twitter on Dgraph
+## Flock 
+
+Flock is twitter Graph model built on Dgraph. 
+
+
+Flock has two parts, 
+- [Mutation program](./main.go)
```

Comment:
Could create this is heading 3/4 whichever is next. Alternatively, you could do the following -
```
- [*Mutation program*](./main.go) - Makes use of the tweets available from twitter developer API's and builds the Graph model on top of Dgraph. Here is the Graph schema of Flock
```